src/ansys/fluent/core/quantity.py

```diff
@@ -717,16 +717,98 @@ def __eq__(self, other):
     def __neg__(self):
         return Quantity(-self.value, self.unit)
 
-    def __ge__(self, other):
-        if (
-            isinstance(other, Quantity)
-            and self.get_dimensions_list() == other.get_dimensions_list()
-            and self._si_unit == other._si_unit
+    def __gt__(self, other):
+        if isinstance(other, Quantity) and (
+            self.get_dimensions_list() != other.get_dimensions_list()
         ):
+            raise ValueError("Incompatible dimensions.")
+        elif isinstance(other, Quantity):
             return float(self) > float(other)
         elif (
             self.is_dimension_less()
             and (not isinstance(other, Quantity))
             and isinstance(other, (float, int))
         ):
             return float(self) > other
+        else:
+            raise TypeError("Incompatible quantities.")
+
+    def __ge__(self, other):
+        if isinstance(other, Quantity) and (
+            self.get_dimensions_list() != other.get_dimensions_list()
+        ):
+            raise ValueError("Incompatible dimensions.")
+        elif isinstance(other, Quantity):
+            return float(self) >= float(other)
+        elif (
+            self.is_dimension_less()
```

Comment:
I think the name should be `is_dimensionless`.

Reply:
Done. Thank you.